### DISCUSSION TITLE
Script Browser Window Suddenly Opening Halfway Off Screen

### DISCUSSION THREAD

**Discussion by carmitrey** [brentvollebregt/auto-py-to-exe#341] [General]
Up until today, this has worked flawlessly and is a great and easy tool for those looking to distribute apps without needing to worry about having end users download dependencies.

However, starting recently, when I click "Browse" next to the Script Location field, the Finder window is opening half-way off the screen and I cannot figure out how to fix it or move it back. I am unable to simply drag it back into view. 

<img width="1535" alt="Screen Shot 2022-12-07 at 2 54 54 PM" src="https://user-images.githubusercontent.com/76013426/206282490-3758f10c-b737-4c4b-981b-e39972aec34b.png">

I have tried simply rebooting Mac OS but that doesn't seem to fix the issue and wasn't sure maybe there is some caching somewhere that keeps loading the window in this position rather than in the center of my screen.

If it's any help, this seemed to start after I had started using an additional monitor and discontinued using it later.

Appreciate any assistance on figuring out how to move this window back into view, and happy to make an issue card if it turns out to be a bug.

Thanks!

**Comment by carmitrey** [comment]
I moved this to an issue, just a heads up.

**Reply by brentvollebregt** [reply]
For others curious: #342

**Reply by carmitrey** [reply]
> For others curious: #342
(Posting here for visibility if anyone else stumbles upon this issue)

Oddly, if I comment out the line `root.wm_attributes('-topmost', 1)`, the window centers perfectly in the middle of my main monitor again.

Within [dialogs.py](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/auto_py_to_exe/dialogs.py) of the auto-py-to-exe package, I commented out lines 28, 51, 62, 73, and it seems to be working again!
